### PR TITLE
Bump rubocop version to 0.53

### PR DESCRIPTION
**Pull request by 0xifis**
* Bump rubocop version to 0.53 to match latest stable version.
* Update .rubocop.yml to fix wrong namespace warning (Lint/EndAlignment -> Layout/EndAlignment)

### CODE REVIEW

**Comment by vishthemenon** [comment]
Issue originated from changes to namespace in the latest stable rubocop version https://github.com/bbatsov/rubocop/issues/4704

File: Gemfile [hunk 1]
```diff
@@ -32,7 +32,7 @@ gem "uglifier", ">= 1.3.0", require: false
 # Explicitly avoid 1.x that doesn't support Ruby 2.4+
 gem "json", ">= 2.0.0"
 
-gem "rubocop", ">= 0.47", require: false
+gem "rubocop", ">= 0.53", require: false
```
Comment (koic) [inline Gemfile]: To make RuboCop version the same between local environment and Code Climate, it also need to change the channel of codeclimate.yml.
https://github.com/rails/rails/blob/master/.codeclimate.yml#L26

However, Code Climate doesn't seem to be ready to make the channel 0.53 yet.
https://github.com/codeclimate/codeclimate-rubocop/issues/118

AFAIK, you will have to wait for that release to synchronize versions.

Comment (vishthemenon) [reply Gemfile]: @koic Thanks for pointing that out. I just realised that earlier today too. A possible interim solution would be to have a separate .rubocop.yml for codeclimate using channel 0.52. But the codeclimate team seems fairly quick with adding new channels so we can just simply wait for them.

**Comment by koic** [comment]
@vishthemenon RuboCop 0.54.0 has been released and this version is now available on Code Climate.
https://github.com/codeclimate/codeclimate-rubocop/issues/119#issuecomment-380231314

Would you update this PR?

**Comment by rafaelfranca** [comment]
Thank you for the pull request. We just updated to 0.54

**Comment by vishthemenon** [comment]
@koic Sorry, i was  bit distracted last couple of weeks. Thank you @rafaelfranca

**Comment by koic** [comment]
@vishthemenon Don't worry, It's OK 😃

**Pull request closed without merging**